formatter/src/main/java/org/togetherjava/formatter/tokenizer/TokenType.java

```diff
@@ -71,6 +72,8 @@
     PLUS("+"),
     MINUSMINUS("--"),
     MINUS("-"),
+    BOOL_AND("&&"),
+    BOOL_OR("||"),
 
     // this has to be other DIVIDE or else it'll be seen as two divides
     COMMENT(Pattern.compile("^(//.*|/\\*[^*]*\\*+(?:[^/*][^*]*\\*+)*/)")),
```

Comment:
A comment can begin in the same line after a statement 2, provided the statement has terminated, but the `^` forbids that